### PR TITLE
missed some tmp dirs for pprof tool

### PR DESCRIPTION
**Pull request by sjug**
<see title>

### CODE REVIEW

File: agent/tool-scripts/pprof [hunk 1]
```diff
@@ -116,12 +122,12 @@ get_masters() {
     while true; do
         i=1
         for master in "${masters[@]}"; do
-            oc get --raw /debug/pprof/profile?seconds=${interval} --server https://${master}:8443 > ${master}_api_profile && ${tool_bin} tool ${tool} -text -seconds=${interval} -output $PPROF_TMPDIR/${master}/$(date +%Y-%m-%d:%H:%M:%S)_api_profile.txt /tmp/${master}_api_profile &
+            oc get --raw /debug/pprof/profile?seconds=${interval} --server https://${master}:8443 > /tmp/${master}_api_profile && ${tool_bin} tool ${tool} -text -seconds=${interval} -output $PPROF_TMPDIR/${master}/$(date +%Y-%m-%d:%H:%M:%S)_api_profile.txt /tmp/${master}_api_profile &
```
Comment (portante) [inline agent/tool-scripts/pprof]: Can we consider doing one of the two choices below?

 * change references to `/tmp` to be `$dir`, changing line 21 as well to set `dir` to be the value of `pbench_tmp`
 * remove the use of `$dir`, and the variable definition at line 21, and use `$pbench_tmp` everywhere?

Comment (sjug) [reply agent/tool-scripts/pprof]: Where is `${pbench_tmp}` declared?

Comment (portante) [reply agent/tool-scripts/pprof]: Is provided by line 8.

**Comment by sjug** [comment]
@portante Anything else in this one?

File: agent/tool-scripts/pprof [hunk 2]
```diff
@@ -114,56 +119,44 @@ collect_data() {
...
+            oc get --raw /debug/pprof/profile?seconds=${interval} --server https://${master}:8443 > ${pbench_tmp}/${master}_api_profile && ${tool_bin} tool ${tool} -text -seconds=${interval} -output ${PPROF_TMPDIR}/${master}/$(date +%Y-%m-%d:%H:%M:%S)_api_profile.txt ${pbench_tmp}/${master}_api_profile &
             master_pids[${master}_api]=$!
-            oc get --raw /debug/pprof/heap --server https://${master}:8443 > ${master}_api_heap && ${tool_bin} tool ${tool} -text -seconds=${interval} -output $PPROF_TMPDIR/${master}/$(date +%Y-%m-%d:%H:%M:%S)_api_heap.txt /tmp/${master}_api_heap &
-            oc get --raw /debug/pprof/profile?seconds=${interval} --server https://${master}:8444 > ${master}_controller_profile && ${tool_bin} tool ${tool} -text -seconds=${interval} -output $PPROF_TMPDIR/${master}/$(date +%Y-%m-%d:%H:%M:%S)_controller_profile.txt /tmp/${master}_controller_profile &
+            oc get --raw /debug/pprof/heap --server https://${master}:8443 > ${pbench_tmp}/${master}_api_heap && ${tool_bin} tool ${tool} -text -seconds=${interval} -output ${PPROF_TMPDIR}/${master}/$(date +%Y-%m-%d:%H:%M:%S)_api_heap.txt ${pbench_tmp}/${master}_api_heap &
```
Comment (portante) [inline agent/tool-scripts/pprof]: It seems we still need to track the created pid here and make sure we wait for below.

File: agent/tool-scripts/pprof [hunk 3]
```diff
@@ -114,56 +119,44 @@ collect_data() {
...
+            oc get --raw /debug/pprof/profile?seconds=${interval} --server https://${master}:8444 > ${pbench_tmp}/${master}_controller_profile && ${tool_bin} tool ${tool} -text -seconds=${interval} -output ${PPROF_TMPDIR}/${master}/$(date +%Y-%m-%d:%H:%M:%S)_controller_profile.txt ${pbench_tmp}/${master}_controller_profile &
             master_pids[${master}_controller]=$!
-            oc get --raw /debug/pprof/heap --server https://${master}:8444 > ${master}_controller_heap && ${tool_bin} tool ${tool} -text -seconds=${interval} -output $PPROF_TMPDIR/${master}/$(date +%Y-%m-%d:%H:%M:%S)_controller_heap.txt /tmp/${master}_controller_heap &
-            if (( i++%${#masters[@]}==0 )); then
-		for pid in ${master_pids[*]}; do
-                    wait ${pid}
-                done
-            fi
+            oc get --raw /debug/pprof/heap --server https://${master}:8444 > ${pbench_tmp}/${master}_controller_heap && ${tool_bin} tool ${tool} -text -seconds=${interval} -output ${PPROF_TMPDIR}/${master}/$(date +%Y-%m-%d:%H:%M:%S)_controller_heap.txt ${pbench_tmp}/${master}_controller_heap &
```
Comment (portante) [inline agent/tool-scripts/pprof]: Seems like we need to track this one as well ...

File: agent/tool-scripts/pprof (hunk 2 above)
Comment (sjug) [reply agent/tool-scripts/pprof]: We now wait for everything

File: agent/tool-scripts/pprof (hunk 3 above)
Comment (sjug) [reply agent/tool-scripts/pprof]: Test

**Pull request merged**